dist/s_all

```diff
@@ -87,6 +87,7 @@ run "./s_clang-format"
 run "python prototypes.py"
 run "sh ./s_typedef -b"
 run "python test_tag.py"
+run "./s_fixme"
```

Comment:
Nitpick: call using `sh ./s_fixme` like the above scripts

Reply:
I'd like to stick to Bash. Let's see if this is ok with the other reviewers.

Reply:
The standard objection to something like this is that using `sh` is more portable. POSIX requires `/bin/sh to be on a system`; it doesn't require `/bin/bash`.  But we have other scripts that use bash, so this seems acceptable to me.  

If you want to keep it this way, I suggest adding a comment that the `s_fixme` scripts requires bash so that somebody doesn't come along in the future and unwittingly change it.

Reply:
Thanks for the details and the suggestion @keitharnoldsmith, changes were done in 04d1d6ea6cbd313a28da17ac423541435acb86fd.